bittensor/commands/delegates.py

```diff
@@ -556,6 +722,16 @@ def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
         with bittensor.__console__.status(":satellite: Loading delegates..."):
             delegates: list[bittensor.DelegateInfo] = subtensor.get_delegates()
 
+            try:
+                prev_delegates = subtensor.get_delegates(max(0, subtensor.block - 1200))
+            except SubstrateRequestException:
+                prev_delegates = None
+
+        if prev_delegates is None:
+            bittensor.__console__.print(
+                ":warning: [yellow]Could not fetch delegates history[/yellow]"
+            )
+
         show_delegates(
             delegates,
             prev_delegates=None,
```

Comment:
Do we need to pass `prev_delegates` here since we are fetching it earlier

Reply:
Nice catch! Thanks